### PR TITLE
81063 - health check

### PR DESCRIPTION
**Pull request by kiran1501497**
81063 Devops- Health Check

### CODE REVIEW

File: src/UKHO.ERPFacade.API/Controllers/HealthCheckController.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class HealthCheckController : BaseController<HealthCheckController>
+    {      
+        public HealthCheckController(IHttpContextAccessor httpContextAccessor) : base(httpContextAccessor)
+        {
+        }
+
+        [HttpPost]
```
Comment (Vishal14583) [inline src/UKHO.ERPFacade.API/Controllers/HealthCheckController.cs:13]:
```suggestion
        [HttpGet]
```

Comment (Vishal14583) [reply src/UKHO.ERPFacade.API/Controllers/HealthCheckController.cs:13]: This should be Get request not Post

File: Deployment/templates/continuous-deployment.yml
Comment (amogh14893) [inline Deployment/templates/continuous-deployment.yml:1]: small correction in DisplayName - Instead ERP Facade **Webapp** , ERP Facade **API EndPoit**

**Review by amogh14893 — CHANGES_REQUESTED** [review]
Minor changes in check service status task , Else changes look good to me

**Review by amogh14893 — APPROVED** [review]

**Review by jrippington — APPROVED** [review]
Looks good to me, thanks!

**Review by jrippington — APPROVED** [review]
Looks good to me, thanks!

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**